### DISCUSSION TITLE
Docker "unhealthy" and server not starting after new update.

### DISCUSSION THREAD

**Discussion by PiotrZadka** [mbround18/valheim-docker#182] [Help!] [answered]
Operating System: Ubuntu 20.04 (LTS) x64
Docker Version: [What docker version, can be found with `docker --version`]
Docker Compose Version: Docker version 20.10.3, build 48d30b5
User & Group On Host: 0:0. If
Are you running with BepInEx? No
![image](https://user-images.githubusercontent.com/26087684/109186320-b45a3380-7788-11eb-940e-ada4c16370af.png)

Basically after the new update the server is no longer visible. I've tried to do **docker pull mbround18/valheim** and do docker-compose up to set up new container but no luck. I bash into container and try to run **odin start** but no success either

**Comment by mbround18** [comment, marked as answer]
Whats the output log?

**Reply by PiotrZadka** [reply]
I'm sorry not sure where to find that one

**Reply by PiotrZadka** [reply]
![image](https://user-images.githubusercontent.com/26087684/109192248-e40c3a00-778e-11eb-90f1-2c80b8821c57.png)

**Reply by mbround18** [reply]
an you give me the full log? This snippet is normal. My thoughts are your server updated and your mods are no longer compatible.

**Reply by PiotrZadka** [reply]
I'm not running any mods just plain container from this repo

**Reply by mbround18** [reply]
can you uninstall by removing all of valheims content? and then start the container again

**Reply by PiotrZadka** [reply]
Removing all valheims content helped. Thanks!

**Reply by mbround18** [reply]
you found a good bug btw 👍 thanks for highlighting it :) should be merged in and on the `:latest` docker image. If you pull a new copy of `:latest` you shouldnt experience the issue again.
